### DISCUSSION TITLE
Need help on running k3s-rootless as a systemctl service

### DISCUSSION THREAD

**Discussion by giriappvx** [k3s-io/k3s#5659] [Q&A]
Have gone though [this](https://rancher.com/docs/k3s/latest/en/advanced/#running-k3s-with-rootless-mode-experimental) and started setting up k3s rootless service & it failed to start. Attached the debug log. Please go through & let me know what am I doing wrong.

OS NAME="Red Hat Enterprise Linux"
VERSION="8.5 (Ootpa)"
ID="rhel"

k3s version v1.23.3+k3s1 (5fb370e5)
go version go1.17.5
 
[k3s-rootless.log](https://github.com/k3s-io/k3s/files/8869739/k3s-rootless.log)

**Comment by brandond** [comment]
The fatal errors are right there in the last three lines of the log:
```
FATA[0006] delegated cgroup v2 controllers are required for rootless. 
FATA[0006] child died: command [k3s --debug server --rootless --snapshotter=fuse-overlayfs] exited: exit status 1 
FATA[0007] child exited: exit status 1    
```

**Reply by giriappvx** [reply]
Thanks for your reply @brandond, Yes.  Do I have to enable cgroup v2 to run k3s as rootless, enabling cgroup v2 is optional (resource limitation) right ?
Here I'm restricted to upgrade systemd version to 244, So I didn't try enabling cgroup v2.

**Reply by brandond** [reply]
As the docs page you linked says:
> Known Issues with Rootless mode
> * Cgroups
>   Cgroup v1 is not supported. v2 is supported.
